shared/js/initApp.js

```diff
@@ -172,21 +172,64 @@ dashboard.saveProject = function(callback) {
       location.hash = dashboard.currentApp.id = data.id;
       callbackSafe(callback, data);
     });
-    dashboard.showProjectHeader();
+    if (!dashboard.currentApp.hidden) {
```

Comment:
Right now `hidden` is tautological to having a `projectTemplateLevelName`, but will that always be the case?  If we start listing embedded projects in the project list, is the expectation that the project header will start appearing for these levels?